docs/posts/threads-launch-twitter.md

```diff
@@ -27,7 +27,7 @@ The man behind Facebook has somehow managed to make the Twitter experience worse
 
 And this is where it gets clever. Say you tire of blue-tick brands shitposting, and want to delete your account? Hard luck. Like an ill-advised tattoo, Threads accounts are effectively permanent. If you delete your Threads account, your Instagram account goes too. You're locked in this shiny pit of brand based flimflam, and your Instagram account is hostage.
 
-Threads is what would happen if Twitter and Instagram made out in a bowling alley. It's all their worst parts combined - but it may well succeed. Rocket-man Musk's changes to Twitter have not exactly made it 'brand friendly'. Threads, meanwhile, is shaping up to be a paradise for in-your-face brands - and the AdTech industry would love for you to join them. As Chris Black [put it](https://www.gq.com/story/pulling-weeds-chris-black-twitter-is-better-than-ever), Threads is just 'another marketing channel masquerading as a community.' When the site inevitably introduces ads, the scared Twitter advertisers will flock. A _TechCrunch_ headline [ran](https://techcrunch.com/2023/07/06/you-cant-post-ass-threads-is-doomed-meta-instagram-twitter/) 'You can’t post ass, Threads is doomed'.  A *TechCrunch* headline [ran](https://techcrunch.com/2023/07/06/you-cant-post-ass-threads-is-doomed-meta-instagram-twitter/) 'You can’t post ass, Threads is doomed'. It should have run: 'You can't post ass, Threads will boom'.
+Threads is what would happen if Twitter and Instagram made out in a bowling alley. It's all their worst parts combined - but it may well succeed. Rocket-man Musk's changes to Twitter have not exactly made it 'brand friendly'. Threads, meanwhile, is shaping up to be a paradise for in-your-face brands - and the AdTech industry would love for you to join them. As Chris Black [put it](https://www.gq.com/story/pulling-weeds-chris-black-twitter-is-better-than-ever), Threads is just 'another marketing channel masquerading as a community.' When the site inevitably introduces ads, the scared Twitter advertisers will flock. A *TechCrunch* headline [ran](https://techcrunch.com/2023/07/06/you-cant-post-ass-threads-is-doomed-meta-instagram-twitter/) 'You can’t post ass, Threads is doomed'. It should have run: 'You can't post ass, Threads will boom'.
```

Comment:
1. A "shiny pit" is an oxymoron. There is no light and therefore cannot be shiny.
2. 'Should have run' is incorrect grammar. 'Run' is present tense and 'ran' is something that happened in the past. Likewise, there are many errors in this whole post, and probably other blog posts. I would also refrain from talking about two apps' "make out" session, as it is slightly inappropriate for a privacy blog. Not to mention that the sentence doesn't make sense in the first place, because that's not how reproduction works. Either way, please refrain from this sort of language if possible. 

Always proofread. :)